package.json

```diff
@@ -67,6 +86,7 @@
     "cross-env": "^7.0.3",
     "electron": "^22.3.4",
     "electron-builder": "^23.6.0",
+    "electron-notarize": "^1.2.2",
```

Comment:
I'm seeing deprecation warnings about `electron-notarize` in favor of `@electron/notarize` (which is already installed). Did you have problems using `@electron/notarize`?

Reply:
Moved to use @electron/notarize.